src/Changelogs/ChangelogDatastore.ts

```diff
@@ -73,7 +73,7 @@ export class ChangelogDataStore {
   async fetchAllChangelogs(params: {
     learningObjectId: string,
   }): Promise<ChangeLogDocument[]> {
-    const changelogs = await this.db
+    return await this.db
```

Comment:
`async await` not needed here if we are just returning the Promise.
```suggestion
    return  this.db
```